src/iuse_actor.cpp

```diff
@@ -2578,6 +2609,17 @@ repair_item_actor::attempt_hint repair_item_actor::repair( player &pl, item &too
         roll = NEUTRAL;
     }
 
+    if (t->has_quality(quality_sew)) {
```

Comment:
That are the remains of the first attempt. I guess we don't really need to check for qualities like this.